pkg/crc/machine/generate_bundle.go

```diff
@@ -31,9 +31,13 @@ func (client *client) GenerateBundle() error {
 	// Stop the cluster
 	currentState, err := client.Stop()
 	if err != nil {
-		return err
+		logging.Debug("Forcefully powering off")
+		if err := client.PowerOff(); err != nil {
+			return err
+		}
```

Comment:
`Force poweroff the VM if not gracefully shutdown` feels like something which deserves its own issue/more discussion. If I'm reading this right, crc can report a successful bundle generation after a forced shutdown? How do we know the image is not corrupted in such a situation? I'm not necessarily against an option to allow forced shutdowns, but I'm not sure we want to do this by default. Can this be extracted to a separate PR?

Reply:
@cfergeau https://github.com/code-ready/crc/issues/2468